spec/Dishes_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'dishes'
+
+describe Dishes do
+
+  describe "#display_menu" do
+    it 'displays the menu and prices' do
+      expect(subject.display_menu).to eq ({ "chow mein" => 2.99, "egg fried rice" => 1.99, "spring rolls" => 2.99 })
+      end
```

Comment:
`end` at 8, 6 is not aligned with `it 'displays the menu and prices' do` at 6, 4